### PR TITLE
Archive this repo with pointer to the new sample

### PR DESCRIPTION
**Pull request by JosueJoshua**
According to the https://github.com/Azure-Samples/storage-blob-resize-function-node-v10/pull/1#issuecomment-788220155,  update README to archive this repo with pointer to the [new sample](https://github.com/azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples).

@ramya-rao-a @jongio for notification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ description: "This sample implements a function triggered by Azure Blob Storage
 
 # Azure Storage Blob Trigger Image Resize Function in Node.js using the v10 SDK
 
+> IMPORTANT! This sample uses the older version of the Storage Blob SDK i.e. azure-storage. Please refer to the sample [storage-blob-resize-function](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples/storage-blob-resize-function) that uses the latest version of the Storage Blob SDK i.e. [@azure/storage-blob](https://www.npmjs.com/package/@azure/storage-blob) instead.
```
Comment (ramya-rao-a) [inline README.md]:
```suggestion
> IMPORTANT! This sample uses the version 10 of the Storage Blob SDK. Please refer to the sample [storage-blob-resize-function](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples/storage-blob-resize-function) that uses the latest version of the Storage Blob SDK instead.
```

**Comment by ramya-rao-a** [comment]
Please wait until the PR that adds the sample to azure-sdk-for-js repo is merged to merge this PR

File: README.md (hunk 1 above)
Comment (zzhxiaofeng) [reply README.md]: @ramya-rao-a , I have updated the readme as you said.

**Comment by ramya-rao-a** [comment]
Based on https://github.com/Azure/azure-sdk-for-js/pull/14074#issuecomment-856380020, we have decided to update the sample in this repo to use v12 storage SDKs. So, closing this PR

**Comment by v-xuto** [comment]
@ramya-rao-a    We have updated this Readme to point to https://github.com/Azure-Samples/storage-blob-resize-function-node.
Because JosueJoshua has left, so we don't have permission to change the title and description.

**Comment by v-xuto** [comment]
Description:
Based on [Azure/azure-sdk-for-js#14074 (comment)](https://github.com/Azure/azure-sdk-for-js/pull/14074#issuecomment-856380020), update this Readme to point to https://github.com/Azure-Samples/storage-blob-resize-function-node.

**Review by ramya-rao-a — APPROVED** [review]

**Pull request merged**